src/common/UnitTests-CommonLib/Settings.Tests.cpp

```diff
@@ -602,221 +602,220 @@ namespace UnitTestsCommonLib
         }
     };
 
-    TEST_CLASS(CustomActionObjectUnitTests)
-    {
-    public:
-        TEST_METHOD(CustomActionObjectName)
-        {
-            const std::wstring json = L"{\"action_name\": \"action name\", \"value\": \"action value\"}";
-            CustomActionObject obj = CustomActionObject::from_json_string(json);
-            Assert::AreEqual(std::wstring(L"action name"), obj.get_name());
-        }
-
-        TEST_METHOD(CustomActionObjectValue)
-        {
-            const std::wstring json = L"{\"action_name\": \"action name\", \"value\": \"action value\"}";
-            CustomActionObject obj = CustomActionObject::from_json_string(json);
-            Assert::AreEqual(std::wstring(L"action value"), obj.get_value());
-        }
-    };
+    TEST_CLASS(CustomActionObjectUnitTests){
+        public:
+            TEST_METHOD(CustomActionObjectName){
```

Comment:
Wrong formatting. Let's skip the test files for now, since the formatter doesn't seem to work correctly.

Reply:
hmm, it seems like there's a workaround for it: https://developercommunity.visualstudio.com/comments/684193/view.html. I'll try it

Reply:
@enricogior  I was able to fix this, PTAL